cdat/cdat/axis_preview.py

```diff
@@ -54,24 +60,5 @@ def resizeEvent(self, ev):
         super(AxisPreviewWidget, self).resizeEvent(ev)
         self.update()
 
-
-if __name__ == "__main__":
-    import vcsaxis
-    from PySide import QtGui, QtCore
-    import cdms2
-
-    app = QtGui.QApplication([])
-    box = vcs.createboxfill()
-    tmpl = vcs.createtemplate()
-    var = cdms2.open(vcs.sample_data + "/clt.nc")("clt")
-    axis = vcsaxis.VCSAxis(box, tmpl, "x1", var)
-    w = QtGui.QWidget()
-    l = QtGui.QVBoxLayout()
-    w.setLayout(l)
-    preview = AxisPreview(axis)
-    l.addWidget(preview)
-    pb = QtGui.QPushButton("Update")
-    l.addWidget(pb)
-    pb.clicked.connect(preview.update)
-    w.show()
-    app.exec_()
+    def setAxisObject(self, axis):
+        self.axis = axis
```

Comment:
Please add a newline to this file at the end... it's for POSIX compliance (you can read up on it [here](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline))